lib/gemojione.rb

```diff
@@ -98,6 +112,22 @@ def self.replace_named_moji_with_images(string)
     safe_string
   end
 
+  def self.replace_ascii_moji_with_images(string)
+    return string unless string
+    unless string.match(index.ascii_moji_regex)
+      return safe_string(string)
+    end
+
+    safe_string = string.dup
```

Comment:
This line concerns me. I understand that if using the `safe_string` method to ensure html_safe version of the input some ASCII like `<3` would not be found. However this presents a security issue if something like the following is processed as input:

```
I <3 emojione <script>(function() { ...doSomeDamage... })</script>
```

The `<3` would be transformed to the appropriate html tag and also the script tag would not be escaped and returned to be evaluated.

If we can work on a solution for this case that would be great. If not, I'm afraid this is a deal breaker for the feature for injection security purposes.